File: alafia-server/src/main/java/es/alafia/server/model/dto/ClientDTO.java
package es.alafia.server.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ClientDTO {
    private String name;
    private String mail;
    private String bookingId;
    private String dinnerTableId;
    private String restaurantId;
    private Boolean confirmed;
    private List<String> testAnswers;
}

File: alafia-web-application/src/app/model/dto/clientDto.ts
export class ClientDto {
  name: string;
  mail: string;
  oldClientId: string;
  bookingId: string;
  dinnerTableId: string;
  restaurantId: string;
  confirmed: boolean;

  constructor(name: string, mail: string, oldClientId: string, bookingId: string, dinnerTableId: string, restaurantId: string, confirmed: boolean) {
    this.name = name;
    this.mail = mail;
    this.oldClientId = oldClientId;
    this.bookingId = bookingId;
    this.dinnerTableId = dinnerTableId;
    this.restaurantId = restaurantId;
    this.confirmed = confirmed;
  }
}
